### PR TITLE
Update core.sys.freebsd.sys.event

### PR DESCRIPTION
**Pull request by Calrama**

### CODE REVIEW

File: src/core/sys/freebsd/sys/event.d [hunk 1]
```diff
@@ -28,10 +30,11 @@ enum
     EVFILT_PROC     =  -5, /* attached to struct proc */
     EVFILT_SIGNAL   =  -6, /* attached to struct proc */
     EVFILT_TIMER    =  -7, /* timers */
-    // EVFILT_NETDEV   =  -8, /* no longer supported */
+    EVFILT_PROCDESC =  -8, /* attached to process descriptors */
```
Comment (nemanja-boric-sociomantic) [inline src/core/sys/freebsd/sys/event.d]: I would keep both and add deprecation note to this one (can one deprecate enum field?) so it get's removed in the next release and not break code immediately.

Comment (Calrama) [inline src/core/sys/freebsd/sys/event.d]: If I changed existing `EVFILT_NETDEV` to `EVFILT_PROVDESC` I would understand that, but `EVFILT_NETDEV` was already commented out and not usable by any code?

Comment (nemanja-boric-sociomantic) [inline src/core/sys/freebsd/sys/event.d]: Ah, I'm sorry, I've missed that it was already commented. Thank you for pointing that out!

File: src/core/sys/freebsd/sys/event.d [hunk 2]
```diff
@@ -15,6 +15,8 @@ module core.sys.freebsd.sys.event;
 
 version (FreeBSD):
 extern (C):
+nothrow:
+@nogc:
```
Comment (MartinNowak) [inline src/core/sys/freebsd/sys/event.d]: Be careful with those in C headers, sometime callback types are declared which would become more restrictive by that and a deprecated overload taking the old callbacks would be in order.

Comment (Calrama) [inline src/core/sys/freebsd/sys/event.d]: With regards to `@nogc` I see your point, but `nothrow`? Is throwing from a D function called by C even defined behaviour?

**Pull request merged**